src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/HostWorkspace/LanguageServerWorkspaceFactory.cs

```diff
@@ -54,7 +54,12 @@ public LanguageServerWorkspaceFactory(
     public async Task InitializeSolutionLevelAnalyzersAsync(ImmutableArray<string> analyzerPaths)
     {
         var references = new List<AnalyzerFileReference>();
-        var analyzerLoader = Workspace.Services.GetRequiredService<IAnalyzerAssemblyLoaderProvider>().GetShadowCopyLoader();
+        var loaderProvider = Workspace.Services.GetRequiredService<IAnalyzerAssemblyLoaderProvider>();
+
+        // Load all analyzers into a fresh shadow copied load context.  In the future, if we want to support reloading
+        // of solution-level analyzer references, we should just need to listen for changes to those analyzer paths and
+        // then call back into this method to update hte solution accordingly.
+        var analyzerLoader = loaderProvider.CreateNewShadowCopyLoader();
```

Comment:
@dibarbet fyi

Reply:
note: this is just for solution-level analyzers.  Project level analyzers will reload automatically.